masterlist.yaml

```
@@ -3752,6 +3752,26 @@ plugins:
         util: 'TES4Edit v4.0.4b'
 
 ###### Character Appearance - NPCs ######
+  - name: 'Alt. Dancer .*\.esp'
```

```suggestion
  - name: 'Alt\. Dancer .*\.esp'
```
Needs to escape the initial dot as well.